samoa-api/src/main/java/org/apache/samoa/moa/streams/AvroFileStream.java

```diff
@@ -140,7 +140,8 @@ protected boolean readNextInstanceFromFile() {
   public void prepareForUseImpl(TaskMonitor monitor, ObjectRepository repository) {
     super.prepareForUseImpl(monitor, repository);
     String filePath = this.avroFileOption.getFile().getAbsolutePath();
-    this.fileSource.init(filePath, AvroFileStream.AVRO_FILE_EXTENSION);
+    //this.fileSource.init(filePath, AvroFileStream.AVRO_FILE_EXTENSION);
+    this.fileSource.init(filePath, null);
```

Comment:
My Avro files do not have extensions. I realize this is not my personal software but it did happen to me and it can happen to others. Why require an extension at all?